Issues with 60 sec TTL

I don't know if this is still an issue with the latest code.

But I received this error when using 60 sec TTL, no issue when reverting back to 30 sec TTL

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x20 pc=0x40175c]

goroutine 8 [running]:
main.heartbeat(0xc200136d20, 0xa)
        /home/ubuntu/docker/builder/tmp/skydock/main.go:126 +0x2fc
created by main.sendService
        /home/ubuntu/docker/builder/tmp/skydock/main.go:179 +0x2b8

goroutine 1 [semacquire]:
sync.runtime_Semacquire(0xc200162068)
        /usr/lib/go/src/pkg/runtime/zsema_linux_amd64.c:165 +0x2e
sync.(*WaitGroup).Wait(0xc200112180)
        /usr/lib/go/src/pkg/sync/waitgroup.go:109 +0xf2
main.main()
        /home/ubuntu/docker/builder/tmp/skydock/main.go:281 +0x5c1

goroutine 2 [syscall]:

goroutine 4 [syscall]:
os/signal.loop()
        /usr/lib/go/src/pkg/os/signal/signal_unix.go:21 +0x1c
created by os/signal.init·1
        /usr/lib/go/src/pkg/os/signal/signal_unix.go:27 +0x2f

goroutine 13 [chan receive]:
main.heartbeat(0xc200136dc0, 0xa)
        /home/ubuntu/docker/builder/tmp/skydock/main.go:116 +0x18e
created by main.sendService
        /home/ubuntu/docker/builder/tmp/skydock/main.go:179 +0x2b8

goroutine 14 [IO wait]:
net.runtime_pollWait(0x7f94aad93e60, 0x72, 0x0)
        /usr/lib/go/src/pkg/runtime/znetpoll_linux_amd64.c:118 +0x82
net.(*pollDesc).WaitRead(0xc2001be3e0, 0xb, 0xc200111930)
        /usr/lib/go/src/pkg/net/fd_poll_runtime.go:75 +0x31
net.(*netFD).Read(0xc2001be360, 0xc20016a000, 0x1000, 0x1000, 0x0, ...)
        /usr/lib/go/src/pkg/net/fd_unix.go:195 +0x2b3
net.(*conn).Read(0xc2001620a0, 0xc20016a000, 0x1000, 0x1000, 0x0, ...)
        /usr/lib/go/src/pkg/net/net.go:123 +0xc3
bufio.(*Reader).fill(0xc2001b6ae0)
        /usr/lib/go/src/pkg/bufio/bufio.go:79 +0x10c
bufio.(*Reader).ReadSlice(0xc2001b6ae0, 0xa, 0x0, 0x0, 0x0, ...)
        /usr/lib/go/src/pkg/bufio/bufio.go:262 +0x202
net/http.readLine(0xc2001b6ae0, 0x0, 0x0, 0x0, 0x0, ...)
        /usr/lib/go/src/pkg/net/http/chunked.go:91 +0x57
net/http.(*chunkedReader).beginChunk(0xc2001bd390)
        /usr/lib/go/src/pkg/net/http/chunked.go:47 +0x28
net/http.(*chunkedReader).Read(0xc2001bd390, 0xc20017be00, 0x200, 0x200, 0x200, ...)
        /usr/lib/go/src/pkg/net/http/chunked.go:65 +0x8f
net/http.(*body).Read(0xc2001bc7c0, 0xc20017be00, 0x200, 0x200, 0x0, ...)
        /usr/lib/go/src/pkg/net/http/transfer.go:535 +0x94
encoding/json.(*Decoder).readValue(0xc2001b5540, 0xc2000b90a0, 0x7f94aad50010, 0x420853)
        /usr/lib/go/src/pkg/encoding/json/stream.go:124 +0x46a
encoding/json.(*Decoder).Decode(0xc2001b5540, 0x6c3700, 0xc2001620c8, 0xc200167570, 0xc2001bc7c0, ...)
        /usr/lib/go/src/pkg/encoding/json/stream.go:44 +0x60
github.com/crosbymichael/skydock/docker.func·002()
        /home/ubuntu/go/src/github.com/crosbymichael/skydock/docker/client.go:186 +0x607
created by github.com/crosbymichael/skydock/docker.(*dockerClient).GetEvents
        /home/ubuntu/go/src/github.com/crosbymichael/skydock/docker/client.go:196 +0xa8

goroutine 15 [chan receive]:
main.eventHandler(0xc2001ccc00, 0xc200112180)
        /home/ubuntu/docker/builder/tmp/skydock/main.go:217 +0x6f
created by main.main
        /home/ubuntu/docker/builder/tmp/skydock/main.go:277 +0x55f

goroutine 16 [chan receive]:
main.eventHandler(0xc2001ccc00, 0xc200112180)
        /home/ubuntu/docker/builder/tmp/skydock/main.go:217 +0x6f
created by main.main
        /home/ubuntu/docker/builder/tmp/skydock/main.go:277 +0x55f

goroutine 17 [chan receive]:
main.eventHandler(0xc2001ccc00, 0xc200112180)
        /home/ubuntu/docker/builder/tmp/skydock/main.go:217 +0x6f
created by main.main
        /home/ubuntu/docker/builder/tmp/skydock/main.go:277 +0x55f

goroutine 18 [chan receive]:
github.com/crosbymichael/skydock/docker.func·001()
        /home/ubuntu/go/src/github.com/crosbymichael/skydock/docker/client.go:174 +0x54
created by github.com/crosbymichael/skydock/docker.func·002
        /home/ubuntu/go/src/github.com/crosbymichael/skydock/docker/client.go:181 +0x551
```

--- a/main.go
+++ b/main.go
@@ -108,11 +108,7 @@ func heartbeat(uuid string) {
 		runningLock.Unlock()
 	}()
 
-	var (
-		errorCount int
-		container  *docker.Container
-	)
-
+	var errorCount int
 	for _ = range time.Tick(time.Duration(beat) * time.Second) {
 		if errorCount > 10 {
 			// if we encountered more than 10 errors just quit
@@ -123,7 +119,7 @@ func heartbeat(uuid string) {
 		// don't fill logs if we have a low beat
 		// may need to do something better here
 		if beat >= 30 {
-			log.Logf(log.INFO, "updating ttl for %s", container.Name)
+			log.Logf(log.INFO, "updating ttl for %s", uuid)
 		}
 
 		if err := updateService(uuid, ttl); err != nil {

